### ISSUE TITLE
No granular progress bar when importing a single 2 GB, GZIP-ed CSV

### ISSUE THREAD

**Issue by marklit** [duckdb/duckdb#12454]
### What happens?

Following on from #12453, When importing a single GZIP'ed CSV via an ``INSERT`` statement, there is no granular progress bar. It jumped to 99% and sits there until the import is finished and then it ticks over to 100%.

The insert statements look like this:

```sql
INSERT INTO trips SELECT * FROM READ_CSV('../taxi/trips_xaa.csv.gz');
INSERT INTO trips SELECT * FROM READ_CSV('../taxi/trips_xab.csv.gz');
INSERT INTO trips SELECT * FROM READ_CSV('../taxi/trips_xac.csv.gz');
```

### To Reproduce

See above.

### OS:

Windows 11

### DuckDB Version:

v1.0.0

### DuckDB Client:

duckdb.exe

### Full Name:

Mark Litwintschik

### Affiliation:

Green Idea

### What is the latest build you tested with? If possible, we recommend testing with the latest nightly build.

I have tested with a stable release

### Did you include all relevant data sets for reproducing the issue?

No - I cannot share the data sets because they are confidential

### Did you include all code required to reproduce the issue?

- [X] Yes, I have

### Did you include all relevant configuration (e.g., CPU architecture, Python version, Linux distribution) to reproduce the issue?

- [X] Yes, I have

**Comment by marklit** [comment]
I'm planning on testing the next stable release of DuckDB to see if this issue is still there.

**Comment by marklit** [comment]
This was fixed in v1.1.1 af39bd0dcf as mentioned in #11197
